Update to Alpine:edge, add configuration scripts

frebib opened the pull request:
TODO: Document available options

frebib committed:
Update to Alpine:edge, add configuration scripts
TODO: Document available options

Adam-Ant reviewed (changes requested):
Looks great, especially the config scripts, thank you. 
A few queries:
- Is there a specific reason to use edge instead of a stable release, especially if we are adding the edge testing repo manually?
-  Is there a reason to be using curl for web requests rather than the built in wget, as we're only doing simple GET requests.
- Can/should the COPY and RUN blocks at the bottom be moved into the existing RUN statement to save on a build step?

Other than those small things looks great!

frebib committed:
Suppress xmlstarlet output

frebib commented:
I tried using alpine:3.6 however `mediainfo` must be in edge/main or edge/commu as it failed to build. It's up to you how we deal with this

I can change to wget- I updated the legacy curl request that existed from before.

The COPY block was at the top however I moved it to the bottom to reduce rebuild times

frebib committed:
Use built-in wget instead of curl

frebib committed:
Print Radarr version at build time

frebib committed:
Fix permissions of binaries and libraries in Radarr

frebib committed:
Update README
